components/Header.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+const Header = styled.h1`
```

Comment:
hmm vielleicht kommt das ja noch weiter unten, aber SEO technisch aufjedenfall, wenn du nur ne h1 als Text-logo verwenden möchtest, diese Komponente dennoch um einen nav Tag und den nav Tag um einen Header Tag wrappen

Reply:
ok hat sich erübrigt

Reply:
vielleicht kannst du das aber trotzdem alles in einer Komponente definieren. Also Header, nav, und dein H1. wäre praktischer. Nur'n Vorschlag